loleaflet/src/control/Control.Tabs.js

```diff
@@ -228,8 +228,8 @@ L.Control.Tabs = L.Control.extend({
 		this._map.showPage();
 	},
 
-	_hideSheet: function() {
-		this._map.hidePage();
+	_hideSheet: function(trigger, key) {
+		this._map.hidePage(key.$trigger[0].innerText);
```

Comment:
I think better to identify tab by number not text (no problems with encoding). It doesn't work for me if used with special characters like "ąłóźð". Probably you can use 'this._tabForContextMenu' which stores the id of a tab page?

Reply:
Thank you for your review :) It's all done now :)